src/pages/api/projects/health.page.ts

```diff
@@ -0,0 +1,154 @@
+import { dbpLog, dbpQueryAll } from 'lib/api/supabase/projects'
```

Comment:
**question (non-blocking)** I feel like we could get away without this health check long-term.

To reduce complexity, couldnt we:
- use subgraph as single source of truth
- cron the `update` method

in the update handler:
- get all projects from subgraph.
- upsert - update any existing project with the new data, insert projects that don't exist

When a project updates their metadata on JBM:
- update the db record for that specific project

Am i missing something?

Reply:
yep, this is exactly how it's working right now (though updating the DB after a project updates metadata is tricky, since there's some latency before the data is available in the subgraph)

we could def drop the health page at some point. but it was built so long ago and still works that we might as well keep it as a sanity check for a while. it's saved some headache a few times now during dev